src/lib/utils/featureFlag.ts

```diff
@@ -0,0 +1,20 @@
+export const featureFlags: string[] = [];
+
+export const isFeatureFlagEnabled = (flag: string) => {
```

Comment:
If we want to avoid the issue of having to wrap this inside an `onMount()`, I think we could just `return false` from this function if `browser` is false.